config/go.d/isc_dhcpd.conf

```diff
@@ -82,10 +82,9 @@
 #
 #
 # [ JOBS ]
-#jobs:
-# - name: lease_example
-#   leases_path: '/path/to/ipv4_leases_file'
-#   pools:
-#       - name : example
-#           networks: "192.168.0.0/24 192.168.1.1-192.168.1.63 2001:0DB8::/64"
-#
+jobs:
+ - name: lease_example
+   leases_path: '/path/to/ipv4_leases_file'
+   pools:
+       - name : example
+           networks: "192.168.0.0/24 192.168.1.1-192.168.1.63 2001:0DB8::/64"
```

Comment:
should be commented out because it is an example, there is no reason to have this job with `leases_path: '/path/to/ipv4_leases_file'` and random pools.

Reply:
I removed the comments, because `k8s_kuberlet.conf` has the final part without comments.

Reply:
it has this predefined job because it makes sense to have it, there are bunch of modules that have predefined jobs.

The thing is sense - does it make sense to have it or net. It makes sense when we expect the job to be running with the default parameters. 

Example:

```yaml
# apache
jobs:
 - name: local
   url: "http://127.0.0.1/server_status?auto"
```

it is not commented out because we want to try `http://127.0.0.1/server_status?auto`, if there is apache is running, we get metrics from it.

---

In `isc_dhcpd` example there is no sense at all, that is why we commented out its jobs, we mainly add them as examples.

Reply:
I got it.